snowdrop-util/src/Snowdrop/Util/Containers.hs

```diff
@@ -49,3 +49,9 @@ toOldestFirst (NewestFirst xs) = OldestFirst $ reverse xs
 
 class IsEmpty x where
   isEmpty :: x -> Bool
+
+instance (Hashable i, Hashable v) => Hashable (Map i v) where
```

Comment:
Are these two instances required for `blokchain-util` to compile? If not, I would move them to `impl` back (as the goal of the task was to place `Hashable` near data types which are defined in snowdrop-xx packages, but here is the case for data type, defined externally)

Reply:
@georgeee  Sorry, misunderstood. It seems that these two instances should be moved back